docs/docs/configuration.md

```diff
@@ -51,6 +51,12 @@ cacheTtl: 86400
 # - default = text
 format: "json"
 
+# - A list of files containing glob patterns to ignore
+#   Set this to [] to disable all ignore files
+# - overridden by passing --ignore-files or --disable-ignore-files
+# - default = [".v8rignore"]
```

Comment:
add a note here that default will change to `[".gitignore", ".v8rignore"]` in v5